src/xenia/base/main_win.cc

```diff
@@ -29,7 +29,7 @@ bool has_console_attached_ = true;
 bool has_console_attached() { return has_console_attached_; }
 
 void AttachConsole() {
-  bool has_console = ::AttachConsole(ATTACH_PARENT_PROCESS) == TRUE;
+  bool has_console = ::AttachConsole(ATTACH_PARENT_PROCESS) != FALSE;
```

Comment:
To elaborate: it's highly unlikely we'll ever get junk in this case. If we do, there's going to be something drastically wrong with things anyway that this check failing to work as intended is the least of worries.